client/src/components/RelatedContainer.jsx

```diff
@@ -15,17 +37,32 @@ export default function RelatedContainer ({id, getOneProduct, getStyles}) {
       return fetchedRelatedProducts;
   }
 
+    async function getAverageRating(productId) {
+      const fetchedReviewMetaData = await axios.get(`/reviews/${productId}/meta`)
+      const ratings = fetchedReviewMetaData.data.ratings
+
+      let total =0;
+      let ratingsTotal=0;
+      let average=0;
+
+      for(let number in ratings) {
```

Comment:
Just a note, I also have an average ratings logic i use in my product information. For future reference maybe we could combine it into 1 and import it from some common folder?

also to cut down on length of the code, you can do something like this btw:

 const averageRating = reviews.reduce((acc, review) => {
    return acc + review.rating
  }, 0) / reviews.length;

And parseInt or .toFixed as you please